arch/sim/src/sim/sim_wifidriver.c

```diff
@@ -176,6 +178,23 @@ struct sim_netdev_s
   uint8_t network_id;           /* for sta, default is 0 */
 };
 
+/* for wireless event */
+
+struct wireless_event_s
+{
+  struct nlmsghdr  hdr;                 /* netlink message header */
```

Comment:
Can we use same style as in above structure definitions?

Reply:
Done